scripts/cxg_admin.py

```diff
@@ -76,6 +76,51 @@ def delete_dataset(ctx, uuid):
             exit(0)
 
 
+@cli.command()
+@click.argument("uuid")
+@click.pass_context
+def tombstone_collection(ctx, uuid):
+    """
+    Tombstones the collection specified by UUID.
+
+    You must first SSH into the target deployment using `make db/tunnel` before running.
+    You must first set DEPLOYMENT_STAGE as an env var before running
+    To run
+    ./scripts/cxg_admin.py --deployment dev tombstone-collection 7edef704-f63a-462c-8636-4bc86a9472bd
+
+    """
+
+    with db_session_manager() as session:
+        collection = Collection.get_collection(session, uuid, include_tombstones=True)
+
+        if not collection:
+            click.echo(f"Collection:{uuid} not found!")
+            exit(0)
+        else:
+            print(
```

Comment:
nit: `click.echo` for consistency. Pretty sure they do the same thing so you could make everything a print if you like :D

Reply:
Ah right, fixed.